### PR TITLE
[move]Use some Move 2024 Syntax to update 'utils' example

### PR DESCRIPTION
**Pull request by shaflow01**
## Description 

Use some Move 2024 Syntax to update 'utils' example.

## Test plan 

test2

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

**Comment by shaflow01** [comment]
@amnn  Waitting for your review

**Comment by amnn** [comment]
Hi @shaflow01, the best way to get PRs such as this reviewed quickly is to add the "devx-move" team as a reviewer!

I will try and take a look today.

**Comment by shaflow01** [comment]
how to add it?

**Comment by shaflow01** [comment]
Could you teach me in detail? I would like to try it out

**Comment by amnn** [comment]
When submitting a PR, you can select the "Reviewers" on the right:

Screenshot 2024-05-07 at 12 14 41 PM

Search for "devx-move" there and select the option that shows up!

File: sui_programmability/examples/utils/tests/safe_tests.move [hunk 1]
```diff
@@ -66,21 +66,20 @@ module utils::safe_tests {
...
+        assert!(withdrawn_coin.value() == initial_funds, 0);
 
         test_utils::destroy(withdrawn_coin);
-        ts::return_to_sender(scenario, cap);
+        scenario.return_to_sender(cap);
         ts::return_shared(safe);
 
-
-        ts::end(scenario_val);
+        scenario_val.end();
```
Comment (amnn) [inline sui_programmability/examples/utils/tests/safe_tests.move:82]: With the introduction of receiver syntax, we no longer need to create a different binding for the `&mut scenario` vs taking the scenario by value -- can you replace:

```move
let mut scenario_val = ts::begin(TEST_SENDER_ADDR);
let scenario = &mut scenario_val;
```

With just:

```move
let mut scenario = ts::begin(TEST_SENDER_ADDR);
```

There may be some other tests that need similar treatment.

**Review by amnn — COMMENTED** [review]
Changes look good to me -- there's a further simplification we can make in the case of test scenario usage -- could you take a look?

**Comment by shaflow01** [comment]
> When submitting a PR, you can select the "Reviewers" on the right:
> 
> Screenshot 2024-05-07 at 12 14 41 PM
> Search for "devx-move" there and select the option that shows up!

63dc2187557d26657868d95f812ab67
I don't seem to have this button.This is what troubles me

**Comment by shaflow01** [comment]
Thank you very much for the answers from amnn and Stefan Mysten. I understand now

**Review by amnn — APPROVED** [review]
Ah that's a shame, sorry for the confusion! As Stefan mentioned, we recently added a system to notify the team, so hopefully we'll be quicker to triage and assign reviewers ourselves :) thanks for contributing!

**Pull request merged**